OpenSim/Moco/MocoGoal/MocoOutputGoal.h

```diff
@@ -56,6 +59,19 @@ class OSIMMOCO_API MocoOutputGoal : public MocoGoal {
         return get_divide_by_mass();
     }
 
+    /** Set the exponent applied to the output value in the integrand. This
+    exponent is applied when minimizing the norm of a vector type output. */
+    void setExponent(int exponent) { set_exponent(exponent); }
+    int getExponent() const { return get_exponent(); }
+
+    /** Set the index to the value to be minimized when a vector type
+    Output is specified. For SpatialVec Outputs, indices 0, 1, and 2
+    refer to the rotational components and indices 3, 4, and 5 refer
+    to the translational components. A value of -1 indicates to
+    minimize the vector norm. */
+    void setOutputIndex(int index) { set_output_index(index); }
+    int getOutputIndex() const { return get_output_index(); }
+
```

Comment:
I'm a bit worried that the decoupling of the vector and index could lead to confusion e.g setting index when not using vector, is it handled the same was as outputs elsewhere?